### PR TITLE
Remove job filtering from triage query, should be in kettle

### PR DESCRIPTION
**Pull request by MushuEE**
Reverting #18325 and replacing with #19038

/hold need to update kettle and verify that the change is working prior to merging. 

/cc @michaelkolber

### CODE REVIEW

File: triage/update_summaries.sh [hunk 1]
```diff
@@ -42,8 +42,7 @@ bq --headless --format=json query --max_rows 1000000 \
   from
     [k8s-gubernator:build.all]
   where
-    timestamp_to_sec(started) > TIMESTAMP_TO_SEC(DATE_ADD(CURRENT_DATE(), -14, 'DAY'))
-    and job != 'ci-kubernetes-coverage-unit'" \
+    timestamp_to_sec(started) > TIMESTAMP_TO_SEC(DATE_ADD(CURRENT_DATE(), -14, 'DAY')) \
```
Comment (michaelkolber) [inline triage/update_summaries.sh:45]: This is missing a closing quote.

```suggestion
    timestamp_to_sec(started) > TIMESTAMP_TO_SEC(DATE_ADD(CURRENT_DATE(), -14, 'DAY'))" \
```

Comment (MushuEE) [reply triage/update_summaries.sh]: thanks

**Review by spiffxp — COMMENTED** [review]
I'm not sure how I feel about this.  Even if we're filtering certain jobs out in kettle going forward, there is noisy data already in this dataset in the past.  You wouldn't want to deploy this until kettle has filtered data out for at least 14 days.

Whereas if we decided there are more noisy jobs that should be filtered out, we could make the change here and have it take effect immediately.

**Comment by michaelkolber** [comment]
I feel like it would make more sense to have it in a single location, regardless of where it is. It's more efficient to have it filtered through kettle, and it's faster through here, but I can definitely see someone in the future trying to figure out why their job isn't appearing in Triage only to find out that it was filtered out in a different place than they thought.

**Pull request merged**